wile.py

```diff
@@ -43,7 +44,12 @@ def wile(obj, directory_url, staging, account_key_path, new_account_key_size, ve
 
     if staging:
         directory_url = 'https://acme-staging.api.letsencrypt.org/directory'
-    account_key = get_or_gen_key(account_key_path, new_account_key_size)
+
+    try:
+        account_key = get_or_gen_key(account_key_path, new_account_key_size)
+    except ValueError as e:
+        logger.error('regarding %s: %s' % (account_key_path, str(e)))
```

Comment:
small nitpick: the `logging` methods do formatting on their own, so `logger.error('...', account_key_path, e)` should suffice.
Also, maybe a slightly more descriptive error message would be better? :wink: